Docs/Installation.md

```diff
@@ -0,0 +1,7 @@
+# Installation and Activation
+1. Copy the folder **GDFirebase** to the project path res://addons/
+2. Open your Project Settings
+3. Go to Plugins
+4. Activate the GDFirebase plugin<br>
+![Plugin Section](/Images/plugins_section.png)
+5. From there, you will have an autoload singleton with the variables Auth, Database, and Firestore; reference it by using Firebase.Auth, etc.
```

Comment:
I would prefer for the time being to cut Firestore out of this. While technically there's *some* implementation, it's so incomplete as to be laughable and unusable. I will add this back in once I've managed to implement it end-to-end.

Reply:
By the way, if I'd put that in there, mind removing it? :P

Reply:
Removed it.